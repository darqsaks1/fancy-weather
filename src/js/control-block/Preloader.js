import React from 'react';

const Preloader = () => (
  <div className="fixed">
    <div className="preloader">
      <div className="loader" />
    </div>
  </div>
);

export default Preloader;
