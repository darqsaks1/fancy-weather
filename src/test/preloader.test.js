import React from 'react';
import Preloader from '../js/control-block/Preloader'
import TestRenderer from 'react-test-renderer';

test('Preloader changed dom tree', () => {
  const component = TestRenderer.create(
    <Preloader />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
