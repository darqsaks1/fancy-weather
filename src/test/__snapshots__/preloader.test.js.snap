// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Preloader changed dom tree 1`] = `
<div
  className="fixed"
>
  <div
    className="preloader"
  >
    <div
      className="loader"
    />
  </div>
</div>
`;

exports[`Preloader changes the class when hovered 1`] = `
<div
  className="fixed"
>
  <div
    className="preloader"
  >
    <div
      className="loader"
    />
  </div>
</div>
`;
